staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -250,6 +275,12 @@ func (s *ServingOptions) ApplyTo(c *server.Config) error {
 		BindAddress: net.JoinHostPort(s.BindAddress.String(), strconv.Itoa(s.BindPort)),
 	}
 
+	var err error
+	privilegedLoopbackToken := uuid.NewRandom().String()
+	if c.LoopbackClientConfig, err = c.InsecureServingInfo.NewSelfClientConfig(privilegedLoopbackToken); err != nil {
```

Comment:
Can/Do we fail in `Config.Complete()` or `Run()` without loopback? then I am fine with this.